_posts/en/newsletters/2022-12-07-newsletter.md

```diff
@@ -0,0 +1,153 @@
+This week's newsletter describes an implementation of ephemeral anchors
+and includes our regular sections with the summary of a Bitcoin Core PR
+Review Club meeting, announcements of new releases and release
+candidates, and descriptions of notable changes to popular Bitcoin
+infrastructure projects.
+
+## News
+
+- **Ephemeral anchors implementation:** Greg Sanders [posted][sanders
+  ephemeral] to the Bitcoin-Dev mailing list that he's implemented his
+  idea for ephemeral anchors (see [Newsletter #223][news223 anchors]).
+  [Anchor outputs][topic anchor outputs] are an existing technique made
+  available by Bitcoin Core [CPFP carve outs][topic cpfp carve out] and
+  used in the LN protocol to ensure that both parties involved in a
+  contract can [CPFP][topic cpfp] fee bump a transaction related to that
+  contract.  Anchor outputs have several downsides---some of them
+  fundamental (see [Newsletter #224][news224 anchors])---but others
+  which can be addressed.
+
+    Ephemeral anchors build on the [v3 transaction relay proposal][topic
+    v3 transaction relay] to allow v3 transactions to include a
+    zero-value output paying a script that is essentially `OP_TRUE`,
+    which permits that transaction to be CPFP fee bumped anyone on the
+    network with a spendable UTXO.  The fee-bumping child transaction
+    can itself be RBF fee bumped by anyone else with a spendable UTXO.
+    In combination with other parts of the v3 transaction relay
+    proposal, it can be hoped that this will eliminate all policy-based
+    concerns about [transaction pinning attacks][topic transaction
+    pinning] against time-sensitive contract protocol transactions.
+
+    Additionally, because anyone can fee bump a transaction containing
+    an ephemeral output, it can be used for contract protocols involving
+    more than two participants.  The existing Bitcoin Core carve out
+    rule only reliably works for two participants and [previous
+    attempts][bitcoin core #18725] to increase it required an
+    arbitrary upper limit on participants.
+
+    Sander's [implementation][bitcoin core #26403] of ephemeral anchors
+    makes it possible to begin testing the idea (along with the other v3
+    transaction relay behaviors previously implemented by that
+    proposal's author).
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME:larryruane
+
+{% include functions/details-list.md
+  q0="FIMXE"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/26265#l-FIXME"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BTCPay 1.7.1][] is the latest release of the most widely used
```

Comment:
I'd suggest using the full name here for consistency and clarity, even though the shorter variant is correct as well.
```suggestion
- [BTCPay Server 1.7.1][] is the latest release of the most widely used
```